cicd/internal/workflows/maven-workflows.go

```diff
@@ -117,15 +117,15 @@ func (*mvnFlags) FailAtTheEnd() string {
 }
 
 func (*mvnFlags) RunIntegrationTests() string {
-	return "-PtemplatesIntegrationTests,splunkDeps"
+	return "-PtemplatesIntegrationTests,splunkDeps,confluent-repo"
```

Comment:
I think GH workflow changes have to be in a separate PR and merged before this one (if they're required for other changes in this PR to work).

Reply:
Created a separate PR: https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1486